<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="13" hostname="LAPTOP-K903MU1U" name="com.webtest.demo.TpShopTest01" tests="15" failures="0" timestamp="2020-12-03T10:42:20 CST" time="6.066" errors="2">
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterPhone" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterPhone -->
  <system-out/>
  <testcase name="doBeforeClass" time="5.997" classname="com.webtest.demo.TpShopTest01">
    <error type="org.openqa.selenium.WebDriverException" message="permission denied
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:53&#039;
System info: host: &#039;LAPTOP-K903MU1U&#039;, ip: &#039;192.168.57.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_171&#039;
Driver info: driver.version: FirefoxDriver
remote stacktrace: ">
      <![CDATA[org.openqa.selenium.WebDriverException: permission denied
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'LAPTOP-K903MU1U', ip: '192.168.57.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_171'
Driver info: driver.version: FirefoxDriver
remote stacktrace: 
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:147)
at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:125)
at com.webtest.core.BaseTest.newWebDriver(Unknown Source)
at com.webtest.core.BaseTest.doBeforeClass(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.TestNG.privateMain(TestNG.java:1371)
at org.testng.TestNG.main(TestNG.java:1335)
]]>
    </error>
  </testcase> <!-- doBeforeClass -->
  <system-out/>
  <testcase name="testLogin" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testLogin -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterPhone" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterPhone -->
  <system-out/>
  <testcase name="testLogin" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testLogin -->
  <system-out/>
  <testcase name="testRegisterPhone" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterPhone -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterEmail" time="0.000" classname="com.webtest.demo.TpShopTest01">
    <skipped/>
  </testcase> <!-- testRegisterEmail -->
  <system-out/>
  <testcase name="testRegisterPhone" time="0.069" classname="com.webtest.demo.TpShopTest01">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
[public void com.webtest.demo.TpShopTest01.testRegisterPhone(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.io.IOException,java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testRegisterPhone([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 19832189982,(java.lang.String) 12345678123456789,(java.lang.String) 12345678123456789,(java.lang.String) 19832102222,(java.lang.String) ]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.webtest.demo.TpShopTest01.testRegisterPhone(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.io.IOException,java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testRegisterPhone([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 19832189982,(java.lang.String) 12345678123456789,(java.lang.String) 12345678123456789,(java.lang.String) 19832102222,(java.lang.String) ]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:905)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.TestNG.privateMain(TestNG.java:1371)
at org.testng.TestNG.main(TestNG.java:1335)
]]>
    </error>
  </testcase> <!-- testRegisterPhone -->
  <system-out/>
</testsuite> <!-- com.webtest.demo.TpShopTest01 -->
